src/Connection.php

```diff
@@ -48,7 +48,7 @@ public function __construct(array $config)
         $dsn = $this->getDsn($config);
 
         // You can pass options directly to the MongoDB constructor
-        $options = Arr::get($config, 'options', []);
+        $options = $config['options'] ?? [];
```

Comment:
This is exactly what was done in `Add::get`. https://github.com/laravel/framework/blob/256f4974a09e24170ceeeb9e573651fd5e1c703e/src/Illuminate/Collections/Arr.php#L322-L324

Reply:
Noted that we're not using dot notation here (per [`Arr::get()` docs](https://laravel.com/docs/10.x/helpers#method-array-get)), so these changes seem sensible.